lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -1562,6 +1565,15 @@ def __init__(
 
         self._recursive = recursive
 
+    @staticmethod
+    def _inject_extra_labels_to_alert_rules(rules: Dict, extra_alert_labels: Dict) -> Dict:
```

Comment:
Consider opening an issue for moving this function into https://github.com/canonical/cos-lib/blob/6cd3769727419f50018994591a90c0024542389b/src/cosl/rules.py#L175

As-is we'd have negative impact on coverage, but at the same time we do not want to repeat the tests.